ge/org.osate.ge.errormodel/src/org/osate/ge/errormodel/ui/swt/TypeTokenListEditorDialog.java

```diff
@@ -150,15 +151,17 @@ protected Control createDialogArea(final Composite parent) {
 			this.addTypesButton.addSelectionListener(new SelectionAdapter() {
 				@Override
 				public void widgetSelected(final SelectionEvent e) {
-					final ErrorTypesSelectorModel selectorModel = new ErrorTypesSelectorModel(editorModel, (t) -> true,
-							Collections.emptyList());
+					final Set<URI> selectedTypeSetURIs = editorModel.getSelectedElements(TypeSet.class)
```

Comment:
It's not entirely clear whether or how this works. We will need to discuss for me to understand things.